.env.example

```
@@ -0,0 +1,17 @@
+NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/
+NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/wizard
+
+DATABASE_URL=""
+
+POSTGRES_HOST=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_DATABASE=
+
+POSTGRES_PRISMA_URL=""
+POSTGRES_URL_NON_UNPOOLED="""
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)